crowbar_framework/app/assets/stylesheets/content/panel.scss

```diff
@@ -0,0 +1,56 @@
+.panel-heading {
+  border-bottom: 0;
+}
+
+.panel .panel-heading h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
+  border: 1px;
+  border-color: #ddd;
+  border-left: 7px;
+  border-radius: 5px;
+  border-style: solid;
+  box-shadow: 1px 1px 0px #eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0px` should be written without units as `0`